server/server.js

```diff
@@ -0,0 +1,28 @@
+//Server setup
+
+const express = require('express');
+const morgan = require('morgan');
+const bodyparser = require('body-parser');
+const cors = require('cors');
+const bcrypt = require('bcryptjs');
+const mongoose = require('mongoose');
+
+const app = express();
+
+const HTTP_PORT = process.env.PORT || 3000;
+const SERVER = app.listen(HTTP_PORT, ()=>{
+    console.log("Listening on port 3000...");
+});
+
+//Mongoose connection
```

Comment:
Please externalize connection properties in a separate file config.js. These include Mongoose connection, port number and a secret phrase. use module.exports to export these properties.